lager/lenses/attr.hpp

```diff
@@ -17,15 +18,14 @@ namespace lenses {
 template <typename Member>
 auto attr(Member member)
 {
-    return zug::comp([member](auto&& f) {
-        return [&, f = LAGER_FWD(f)](auto&& p) {
-            return f(LAGER_FWD(p).*member)([&](auto&& x) {
-                auto r    = LAGER_FWD(p);
-                r.*member = LAGER_FWD(x);
-                return r;
-            });
-        };
-    });
+    return getset(
```

Comment:
Hi, @arximboldi!

I had to rewrite `attr` into (the fixed version of) `getset`, because otherwise it also caused double-move when nested and passed with a prvalue. Basically, a code like that is enough to crash it:

```cpp
auto birthday_lens = attr(&debug_person::birthday);
auto person_lens = attr(&debug_freelancer::person);
auto freelancer_birthday_lens = person_lens | birthday_lens;

auto birthday =
    set(freelancer_birthday_lens,
        debug_freelancer(freelancer_copy_info, person_copy_info, birthday_copy_info),
        debug_yearday(birthday_copy_info));
```

I have a weird feeling like **all** the lenses in lager have to be rewritten like that... :( Not only they do double-move, capturing functors into immutable lambdas prevents proper forwarding of the identity functors in `view` and `set`.